### PR TITLE
Display scancodes description and state on a grid

### PR DESCRIPTION
**Pull request by kimci86**
screenshot

### CODE REVIEW

**Review by eXpl0it3r — APPROVED** [review]
Totally forgot about fully reviewing this 😅 

I love it! 🙂 

I guess the only "issue" is that, this is no longer a "simple" example and might not necessarily be fit for inclusion with SFML itself 🤔

File: src/KeyboardView.cpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            m_bloatFactor[event.key.scancode] = 1.5f;
+    }
+}
+
+void KeyboardView::update(sf::Time frameTime)
+{
+    const auto transitionDuration = sf::seconds(0.3f);
+    for (auto& zoom : m_bloatFactor)
+    {
+        const auto abBloatChange = std::min(std::abs(zoom - 1.f), frameTime / transitionDuration);
```
Comment (kimci86) [inline src/KeyboardView.cpp]: typo here (I meant `absBloatChange`), will fix.

**Comment by kimci86** [comment]
> I guess the only "issue" is that, this is no longer a "simple" example and might not necessarily be fit for inclusion with SFML itself :thinking:

Yes I agree. Maybe we can simplify the program later to make it more suitable.

**Comment by eXpl0it3r** [comment]
🥳

**Pull request merged**